tests/curl/Makefile

```diff
@@ -25,10 +18,10 @@ all:
 	$(MAKE) rootfs
 
 tests:
-	$(RUNTEST) $(MYST_EXEC) $(OPTS) --mount /etc/resolv.conf=/etc/resolv.conf rootfs --app-config-path config1.json /curl https://www.microsoft.com 2>&1 > /dev/null
+	$(RUNTEST) $(MYST_EXEC) $(OPTS) rootfs --app-config-path config1.json /curl https://www.microsoft.com 2>&1 > /dev/null
```

Comment:
I don't think we need `--app-config-path config1.json` anymore.

Reply:
Right, changed.